### PR TITLE
fix githook script

### PR DESCRIPTION
**Pull request by lucasdze**
This PR is created to fix the githook issue new hired memeber are getting while setting up their local environment repo
Test Steps:
1. *Include steps to test these changes*

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/build.gradle.kts [hunk 1]
```diff
@@ -810,4 +810,4 @@ dependencies {
     testImplementation("com.willowtreeapps.assertk:assertk-jvm:0.25")
     testImplementation("io.ktor:ktor-client-mock:$ktorVersion")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.9.1")
-}
+}
```
Comment (snesm) [inline prime-router/build.gradle.kts:813]: This probably should not be part of the PR (although testing the hooks might have done this).

File: .environment/githooks.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 pushd "$(dirname "${0}")" 1>/dev/null 2>&1
 
 REPO_ROOT="$(pwd)/.."
-dir=${REPO_ROOT}/.git/hooks/pre-commit
+dir=${REPO_ROOT}/.git/hooks
```
[the remark this answers is not in the document]
Comment (lucasdze) [reply .environment/githooks.sh:7]: yes that actually matters. In order for git to run the githook, the script must be named pre-commit (not a directory). This is actually why it took me longer to understand and fix it.

**Review by snesm — APPROVED** [review]
